processes/app_release.md

```diff
@@ -33,6 +33,9 @@
 
 13. We should also push it to the release `git push release stablex.x --tags`
 
+    13.1 When you push the release for the first time, you might not have the release git repo, if you run the command and the error ` 'release' does not appear to be a git repository` popsup you need to run this command: `git remote add release git@github.com:nextcloud-releases/AppName.git`
```

Comment:
I would move that step into a prerequisite list

Reply:
> GretaD marked this conversation as resolved. 

you only changed the indentation, right?

Reply:
yes, and changed the number from 13.1 to 14.
so from 
1
  1.2
i did
1
2
did i get something wrong?

Reply:
yes, and changed the number from 13.1 to 14.
so from 
1
  1.2
i did
1
2
did i get something wrong?

Reply:
https://docs.nextcloud.com/server/stable/admin_manual/installation/source_installation.html#prerequisites-for-manual-installation

It should be a separate list on the top

```

## Prerequisites
* git `release` origin, e.g. `git@github.com:nextcloud-release/mail.git`

## Process
1. ...
```

Reply:
and today i learned something :)